testing/python/raises.py

```diff
@@ -33,6 +33,18 @@ def __call__(self):
         except pytest.raises.Exception:
             pass
 
+    def test_raises_repr_inflight(self):
+        with pytest.raises(RuntimeError) as excinfo:
```

Comment:
I'd be more comfortable with a very specific exception type here such this doesn't accidentally skip the important bits (for example if `repr` or `pprint` raised a `RuntimeError` the test would silent-pass right now)

```python
class E(Exception): pass

with pytest.raises(E):
    ...

```

should be sufficient